client/src/components/lower-info-area/lower-container.vue

```diff
@@ -125,43 +133,50 @@ export default class LowerContainer extends Vue {
 
 </script>
 
-<style lang="sass" scoped>
-@import "@/assets/scss/theme.scss"
+<style lang="scss" scoped>
+@import "@/assets/scss/theme.scss";
 
-li
-  font-size: 16px
+li {
+  font-size: 16px;
+}
 
-#lower-container
-  padding: 0
-  margin: 0
-  font-size: 16px
-  color: $text
-  background-color: #F1F3F5
+#lower-container {
+  padding: 0;
+  margin: 0;
+  font-size: 16px;
+  color: $gray7;
+  background-color: #F1F3F5;
 
-  .contact-section
-    background-color: $BCgovBlue5
-    color: white
+  .contact-section {
+    background-color: $BCgovBlue5;
+    color: white;
 
-    .contact-info-list
-      margin: 0
-      padding: 0
-      list-style-type: none
+    .contact-info-list {
+      margin: 0;
+      padding: 0;
+      list-style-type: none;
+    }
 
-.info-underline
-  font-size: 1.25rem
-  border-bottom: 1px solid $BCgovBlue3
-  color: white
+    .info-underline {
+      font-size: 1.25rem;
+      border-bottom: 1px solid $BCgovBlue3;
+      color: white;
+    }
+  }
 
-.launch-icon
-  font-size: 16px
-  color: $link
-  padding-left: 4px
+  .square-card {
+    height: 260px;
+    width: 300px;
+    padding: 15px 30px 30px 15px;
+    border-radius: 4px;
+    background-color: white;
+    box-shadow: 2px 2px 4px 0 darkgray;
 
-.square-card
-  height: 260px
-  width: 260px
-  padding: 15px 30px 30px 15px
-  border-radius: 4px
-  background-color: white
-  box-shadow: 2px 2px 4px 0 darkgray
+    .launch-icon {
+      font-size: 16px;
+      color: $link;
+      padding-left: 4px;
```

Comment:
No action required but Vuetify helper class equivalent is `pl-1`. I'm pretty sure each number is the equivalent of 4px, i think*